### DISCUSSION TITLE
Is it possible to filter summary based on ids ?

### DISCUSSION THREAD

**Discussion by aniketgm** [GothenburgBitFactory/timewarrior#555] [Q&A] [answered]
hello,
I know we have `:ids`, however that is to show IDs. What I wish to do is filter based on an ID ?
So say, when I do `timew summary :ids`, I get multiple IDs, '@1', '@2', etc. I want to filter only those with '@1'. Is it possible ?

**Comment by lauft** [comment, marked as answer]
I assume you want something like `timew summary @1 @3 @5`, showing only the intervals with the specified ids. 🤔

Currently, the summary command does not support filtering by ids. Only tags and ranges.

**Reply by aniketgm** [reply]
> I assume you want something like `timew summary @1 @3 @5`, showing only the intervals with the specified ids.

Yes, thanks. But for now, seems like I might have to do some tweaking (script, etc.) to get that behavior.
Adding to my previous question --
Is it possible ? Maybe not now but in future Is it possible to add that ? I can open a feature request issue, if yes ?

**Reply by lauft** [reply]
Adding this should be easy, and fit into the general behavior of Timewarrior. So: yes 🙂
